server/src/main/java/com/exacaster/lighter/rest/MicronautStatementCreationResultToResponseMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.exacaster.lighter.rest;
```

Comment:
Maybe it would make sense to put it in `mappers` package, to differentiate it from controllers?

Reply:
can do